### DISCUSSION TITLE
FastApi doesn't see schema (BaseModel?)

### DISCUSSION THREAD

**Discussion by novastar88** [fastapi/fastapi#6152] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from pydantic import BaseModel

class SimpleText(BaseModel):
    text = str

app = FastAPI()

origins = ["*"]

app.add_middleware(
    CORSMiddleware,
    allow_origins=origins,
    allow_credentials=True,
    allow_methods=["*"],
    allow_headers=["*"],
)


@app.put("/known_words")
async def change_known_words(data: SimpleText):
    return data
```

### Description

FastApi doesn't see any schema in my BaseModel (not only in the docs but in  requests too). It seems that for RestApi schema is empty which is not true. I experimented by adding more fields and types but same effect.

### Operating System

Windows

### FastAPI Version

0.73.0

### Python Version

3.10.1

### Additional Context

As in title, I searched for solution entire web and no luck.
There is no schema in docs:
[1](https://preview.redd.it/7kez2y0rkvt91.png?width=1426&format=png&auto=webp&s=3a536fdff02e86261eff05d0b8da4e6c3121c027)
[2](https://preview.redd.it/91kv3l7skvt91.png?width=1421&format=png&auto=webp&s=227471e11fe7945cf5af4a62f720685cb445bc28)

There is no specified schema after request is made too, here `SimpleText.schema()` returned after making random request like `{'text': 'fdfd'}`:
`{  "title": "SimpleText",  "type": "object",  "properties": {}   }`

I also checked and nothing is returned to my function in the backend after request is made, when I try to use `data.text` I getting empty string (default value)

Help please.

**Comment by odiseo0** [comment, marked as answer]
Because your schema is not well defined.

You wrote:
```py
class SimpleText(BaseModel):
    text = str
```
It should be:
```py
class SimpleText(BaseModel):
    text: str
```

**Comment by novastar88** [comment]
I must admit it's quite embarrassing situation, but now everything works as it should. Thank you, I really need to take a break.

**Comment by tiangolo** [comment]
Thanks for reporting back and closing the issue @novastar88 ! ☕
